server/cli/src/backup/README.md

```diff
@@ -0,0 +1,43 @@
+# Backup and Restore
+
+omSupply server can be backed up and restored via cli. 
+
+`NOTE` In production backups should only be done for omSupply central servers, remote sites are backed up via synchronisation
+
+### Backup
+
+To backup simply run: 
+
+**In development mode**
+
+`cargo run --bin remote_server_cli -- backup` or for postgres `cargo run --bin remote_server_cli --features postgres -- backup`
+
+**In production**
+
+`omSupply-cli backup`
+
+You will need to specify a backup folder in the configuration `.yaml` files - to get started, see the `example.yaml`. Each time backup runs a new folder will be created with this format `D[YYYY]_[mm]_[dd]T[HH]_[MM]_[SS]` e.g. `D2024_08_22T05_05_16`. A successful backup will print new backup name to console.
+
+Backup will contain a folder with all of the app_data (plugins, static files, etc..) and a folder with either sqlite files or postgres database dump. 
+
+### Restore
+
+To restore run: 
+
+**In development mode**
+
+`cargo run --bin remote_server_cli -- restore -b D2024_08_22T05_05_16 -s` or for postgres `cargo run --bin remote_server_cli --features postgres -- restore -b D2024_08_22T05_05_16 -s`
```

Comment:
Also, do you think you should explain what the `-s` flag does? I realised it was for "skip confirmation" with some trial and error, but might be good to be thorough here.

Or... can the flag be more explicit? (i.e `-skip-confirmation`)?

Reply:
I was hoping that more detailed and up to date info can be found with `--help`, added: 97a1bc76ada567de0a4e93481496eac8a8e83aea